logging.adoc

```diff
@@ -0,0 +1,384 @@
+= OTLP Output
```

Comment:
I don't think the information provided is in the direction of what we wish to document.  We want a reference document that identifies the attributes we will push with a log collector and the source of those attributes.  There may be some space to explicitly state the information here is only when using the CLO collector impl but I would otherwise refrain from mentioning CLO or vector at all.

@pavolloffay isn't this repo intended to describe specifics about the payloads we push for the various signals, and not necessarily what or how it is being pushed?